mods/ra2/rules/allied-structures.yaml

```diff
@@ -349,6 +349,20 @@ gaairc:
 	SpawnSurvivors:
 		DeathTypes: ExplosionDeath, BulletDeath
 		Actors: e1, e1
+	SpawnNeighboringActors:
+		ActorTypes: gapad
+		Locations: 1,0, 1,1, 2,1
+
+gapad:
+	Interactable:
+	AlwaysVisible:
+	Reservable:
+	Building:
+		Footprint: x
+		Dimensions: 1,1
+		AllowInvalidPlacement: true
```

Comment:
Why?

Reply:
So it can be placed on top of the main building.